cli/src/wallet.rs

```diff
@@ -447,6 +510,54 @@ pub fn parse_transfer(
     })
 }
 
+pub fn parse_sign_offchain_message(
+    matches: &ArgMatches<'_>,
+    default_signer: &DefaultSigner,
+    wallet_manager: &mut Option<Arc<RemoteWalletManager>>,
+) -> Result<CliCommandInfo, CliError> {
+    let version: u8 = value_of(matches, "version").unwrap_or(0);
```

Comment:
Since you've given `version` a default and already verified type, you can just unwrap here. If someone removes the default in the future, I'd actually rather panic here than fill in zero in a place that may not get visibility.

Reply:
changed